### ISSUE TITLE
[Feature Request] Explained Variance

### ISSUE THREAD

**Issue by NTR0314** [pytorch/rl#3009]
## Motivation

Add Explained Variance logging for PPO. (see https://scikit-learn.org/stable/modules/generated/sklearn.metrics.explained_variance_score.html)

## Solution

The returned tensordict of PPO loss contains Explained Variance as metadata. This can be toggled (default = True) when instantiating the PPO loss.

## Checklist

- [ x ] I have checked that there is no similar issue in the repo (**required**)

**Comment by thomasbbrunner** [comment]
This is done?

https://github.com/pytorch/rl/pull/3010

**Comment by vmoens** [comment]
Yep indeed
